src/promptflow/promptflow/contracts/tool.py

```diff
@@ -285,6 +285,7 @@ def _deserialize_type(v):
             data.get("default", ""),
             data.get("description", ""),
             data.get("enum", []),
+            data.get("custom_type", []),
```

Comment:
Does this `custom_type` work only for connection？

Reply:
Yes, updated the name to custom_type_conn